src/pycropml/pparse.py

```diff
@@ -209,30 +209,45 @@ def Tests(self, elts):
         """ Tests (Test)
         """
         print('Tests')
-
+        modeltest_copy = self._model.tests
```

Comment:
Be carefull. In Python you do not make a copy, but just a reference. If you change modeltest_copy, you will also change self._models.tests.
If you want to make a copy, use
from copy import copy
copy(self._model.tests)